fe/fe-core/src/main/java/com/starrocks/load/loadv2/BulkLoadJob.java

```diff
@@ -84,6 +84,8 @@ public abstract class BulkLoadJob extends LoadJob {
     // we persist these sessionVariables due to the session is not available when replaying the job.
     protected Map<String, String> sessionVariables = Maps.newHashMap();
 
+    protected static final String PRIORITY_SESSION_VARIABLE_KEY = "priority.session.variable.key";
```

Comment:
What's the purpose of this variable? If it's a session variable, it's better to move it to `SessionVariable.java`

Reply:
just for store priority value in bdbje, we can not add new field in journal log. so that use session variables.